tests/conftest.py

```diff
@@ -66,6 +66,23 @@ class LDLfFixtures:
         "([a]tt | [b]tt) <-> ([a | b]tt)",
     ]
 
+    ldlf_advanced_theorems = [
+        "(<a>({f1}) & <a>({f2})) <-> (<a>(({f1}) & ({f2})))",
+        "(<a>({f1}) | <a>({f2})) <-> (<a>(({f1}) | ({f2})))",
+        "([a]({f1}) & [a]({f2})) <-> ([a](({f1}) & ({f2})))",
+        "([a]({f1}) | [a]({f2})) <-> ([a](({f1}) | ({f2})))",
+        "(<a>({f1}) & <b>({f2})) <-> (<a & b>(({f1}) & ({f2})))",
+        "([a]({f1}) | [b]({f2})) <-> ([a & b](({f1}) | ({f2})))",
+        "(<a>({f1}) & [b]({f2})) <-> (<a & !b>({f1}) | <a>(({f1}) & ({f2})))",
+        "([a]({f1}) | <b>({f2})) <-> ([a & !b]({f1}) & [a](({f1}) | ({f2})))",
+        "(<a>({f1}) | <b>({f1})) <-> (<a | b>({f1}))",
+        "([a]({f1}) & [b]({f1})) <-> ([a | b]({f1}))",
+        "<a>({f1}) <-> ([!a]ff & <a>({f1}))",
+        "[a]({f1}) <-> (<!a>tt | [a]({f1}))",
```

Comment:
I think the first of the two is always satisfied regardless of the formulae, because the same term appear twice.
Maybe the second is different. Could we change the two rightmost `a` with `true` to really use `f1` as test?
Double check.

Reply:
No, I think I was wrong. Both have one direction of the implication which does not cancels out. `-> [!a]ff` for the first, and `<- <!a>tt` for the second.